sparse/rest/server.go

```diff
@@ -0,0 +1,27 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/sparse-tools/sparse"
+)
+
+type SyncServer struct {
+	filePath      string
+	holeIntervals []sparse.Interval
+	dataIntervals []sparse.Interval
+	fileIo        sparse.FileIoProcessor
+}
+
+// TestServer daemon serves only one connection for each test then exits
+func TestServer(addr sparse.TCPEndPoint, timeout int) {
```

Comment:
`sparse.TCPEndpoint` is a pretty useless data structure.  Just pass around an `address` string.